src/components/fields/DoubleField.js

```diff
@@ -2,12 +2,12 @@ import React, { PropTypes } from "react";
 import { Double } from 'bson';
 import BsonNumberFieldHOC from './BsonNumberFieldHOC';
 
-function stringToBsonNumber (string) {
+function stringToBsonNumber (string, {decimalPlaces = 2}) {
   const float = parseFloat(string);
   if (isNaN(float)) {
     return undefined;
   } else {
-    return new Double(parseFloat(float.toFixed(2)));
+    return new Double(float.toFixed(decimalPlaces));
```

Comment:
toFixed converts to a string.
It's better we use number to create a Double.
It's not guaranteed that Double will convert string to number.